client/blocks/reader-full-post/_content.scss

```diff
@@ -197,6 +197,29 @@
 			}
 		}
 	}
+
+	sup, sub {
+		vertical-align: baseline;
+		position: relative;
+		font-size: 0.83em;
+	}
+
+	sup {
+		top: -0.4em;
+	}
+
+	sub {
+		bottom: -0.2em;
+	}
+
+	table th,
```

Comment:
These `table` and `img` rules were (accidentally?) scoped to Discover only, but I think they originally applied to all content, so I moved them along with the `sup` and `sub` styles.